sdf/final-project/presentation.md

```diff
@@ -67,8 +79,8 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
 
 ## Time
 - [ ] **Adherence to Time Limit** (2pts): Does the speaker effectively manage their time, completing their presentation within the allotted time frame (3-5 minutes), demonstrating respect for the audience's time and attention?
-  - 1 pt for < 5:30 or > 2:30
-  - 0 for > 5:30
+  - 1 pt for (5-5:30) or (2:30-3)
```

Comment:
nitpik: The points were a bit confusing to me here. Maybe we can put:

**Adherence to Time Limit**: Does the speaker effectively manage their time, completing their presentation within the allotted time frame (3-5 minutes), demonstrating respect for the audience's time and attention?
[ ] (2pts): 3-5 minutes
[ ] (1pt): 2:30-3:00 minutes or 5:00-5:30 minutes
[ ] (0pts): less than 2:30 or more than 5:30